src/main/java/application/controllers/GraphController.java

```diff
@@ -177,21 +202,23 @@ public void initialize(URL location, ResourceBundle resources) {
      * @param depth the depth to draw.
      */
     public void update(Object ref, int depth) {
+        int size = graph.getLevelMaps().size();
+        int min = (int) (drawFrom - screenSize.getMaxX() * 0);
```

Comment:
screenSize.getMaxX() * 0 will always be 0, this has to be simplified to avoid confusion.